network-store-server/src/main/resources/db/changelog/changesets/changelog_20240808T110000Z.xml

```diff
@@ -0,0 +1,62 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                   https://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-latest.xsd">
+
+    <changeSet id="0" author="lesoteti">
+        <createTable tableName="regulationpoint">
+            <column name="networkuuid" type="UUID">
+                <constraints nullable="false"/>
+            </column>
+            <column name="variantnum" type="INTEGER">
+                <constraints nullable="false"/>
+            </column>
+            <column name="regulatedequipmentid" type="VARCHAR(255)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="equipmenttype" type="VARCHAR(255)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="regulationmode" type="VARCHAR(50)"/>
+            <column name="localterminalconnectableid" type="VARCHAR(255)"/>
+            <column name="localterminalside" type="VARCHAR(50)"/>
+            <column name="regulatingterminalconnectableid" type="VARCHAR(255)"/>
+            <column name="regulatingterminalside" type="VARCHAR(50)"/>
+        </createTable>
+        <createIndex indexName="regulationpoint_networkuuid_variantnum_equipmenttype_idx" tableName="regulationpoint">
+            <column name="networkuuid"/>
+            <column name="variantnum"/>
+            <column name="equipmenttype"/>
+        </createIndex>
+        <addPrimaryKey columnNames="networkuuid, variantnum, regulatedequipmentid, equipmenttype" constraintName="regulationpoint_pkey" tableName="regulationpoint"/>
+        <addColumn tableName="staticvarcompensator">
+            <column name="regulationpoint" type="TEXT"/>
+        </addColumn>
+        <addColumn tableName="generator">
+            <column name="regulationpoint" type="TEXT"/>
+        </addColumn>
+        <addColumn tableName="shuntcompensator">
+            <column name="regulationpoint" type="TEXT"/>
+        </addColumn>
+        <addColumn tableName="vscconverterstation">
+            <column name="regulationpoint" type="TEXT"/>
+        </addColumn>
```

Comment:
Why all regulationpoint columns in staticvarcompensator, generator, shuntcompensator and vscconverterstation tables ?

Reply:
my mistake